recipes/apc.rb

```diff
@@ -37,12 +37,8 @@
   action :install
 end
 
-directory '/etc/php.d'
+conf = { 'extension' => 'apc.so' }
```

Comment:
Why can't this be included in the default attribute instead of here?

Reply:
It was more explicit. Just wanted to give it a name. Not really necessary at all

Reply:
Also, I feel like node attributes vary over the lifetime of a node, but this should remain constant :man_shrugging: but it would simplify the code. Should I just go ahead and put into the attributes?

Reply:
I'd say move it to attributes so that we can change it as needed later.

Reply:
Moved it to attributes/apc.rb